javascript/computer_science/project_binary_search_trees.md

```diff
@@ -66,3 +67,13 @@ Write a simple driver script that does the following:
 1. Print out all elements in level, pre, post, and in order
 
 </div>
+
+### Additional Resources
+
+This section contains helpful links to related content. It isn't required, so consider it supplemental.
+
+If you need more help on the three types of binary search tree traversal algorithms, these video resources may aid you in visualizing how a call stack looks when traversing the tree:
+
+- [Inorder Traversal Algorithm](https://www.youtube.com/watch?v=4_UDUj1j1KQ&t=1s) of a binary search tree by Yicheng Gong
+- [Post-order Traversal Algorithm](https://www.youtube.com/watch?v=4Xo-GtBiQN0) of a binary search tree by Yicheng Gong
+- [Pre-order Traversal Algorithm](https://www.youtube.com/watch?v=8xue-ZBlTKQ&ab_channel=ygongcode) of a binary search tree by Yicheng Gong
```

Comment:
This does not follow our layout style guide for Additional Resources. The explanation before 3 bullet points do not allow for additional bullet points at a later time. 
https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md

In addition, the 3 bullet points are repetitive, so I think we should try and condense it down to one bullet point that contains all 3 links. Something like:

```suggestion
- Yicheng Gong has some excellent videos that help visualize the call stack when traversing binary search trees: [Inorder Traversal Algorithm](https://www.youtube.com/watch?v=4_UDUj1j1KQ&t=1s), [Post-order Traversal Algorithm](https://www.youtube.com/watch?v=4Xo-GtBiQN0), and [Pre-order Traversal Algorithm](https://www.youtube.com/watch?v=8xue-ZBlTKQ). 
```

Reply:
We might be able to condense the links even more, something like:
```suggestion
- Yicheng Gong has some excellent videos that help visualize the call stack when traversing binary search trees: [Inorder](https://www.youtube.com/watch?v=4_UDUj1j1KQ&t=1s), [Post-order](https://www.youtube.com/watch?v=4Xo-GtBiQN0), and [Pre-order](https://www.youtube.com/watch?v=8xue-ZBlTKQ) Traversal Algorithms.
```